src/integrationTest/java/uk/gov/hmcts/reform/bulkscan/orchestrator/tasks/CleanupEnvelopesDlqTaskTest.java

```diff
@@ -0,0 +1,48 @@
+package uk.gov.hmcts.reform.bulkscan.orchestrator.tasks;
+
+import com.microsoft.azure.servicebus.IMessageReceiver;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.test.context.junit4.SpringRunner;
+import uk.gov.hmcts.reform.bulkscan.orchestrator.config.IntegrationTest;
+
+import java.time.Duration;
+import java.util.concurrent.TimeUnit;
+import java.util.function.Supplier;
+
+import static org.awaitility.Awaitility.await;
+import static org.mockito.Mockito.verify;
+
+@RunWith(SpringRunner.class)
+@IntegrationTest
+public class CleanupEnvelopesDlqTaskTest {
+
+    @Autowired
+    Supplier<IMessageReceiver> dlqReceiverProvider;
+
+    private CleanupEnvelopesDlqTask dlqTask;
+
+    @Value("${scheduling.task.delete-envelopes-dlq-messages.ttl}")
+    private Duration ttl;
+
+    @Before
+    public void setUp() {
+        dlqTask = new CleanupEnvelopesDlqTask(dlqReceiverProvider, ttl);
```

Comment:
There is no need to keep task in the test. It will be registered in application and schedule will kick off whenever it's ready so the test below will still validate